src/components/SingleMovie/SingleMovie.js

```diff
@@ -1,42 +1,57 @@
 import PropTypes from 'prop-types'
 import './SingleMovie.css';
 import MovieCover from '../MovieCover/MovieCover';
+import Trailer from '../Trailer/Trailer';
 import { useParams } from 'react-router-dom'
 import { useState, useEffect } from 'react';
-import { getSingleMovie } from '../../apiCalls';
+import { getMovieVideos, getSingleMovie } from '../../apiCalls';
 
 
 const SingleMovie = ({fetchApi, setError}) => {
   const [singleMovie, setSingleMovie] = useState(null);
+  const [movieVideos, setMovieVideos] = useState([]);
+  const [trailer, setTrailer] = useState(null);
   const {id} = useParams()
 
-useEffect(() => {
-  fetchApi(getSingleMovie, setSingleMovie, id)
+  useEffect(() => {
+    fetchApi(getSingleMovie, setSingleMovie, id)
+    fetchApi(getMovieVideos, setMovieVideos, id)
+
+    return () => {
+      setError('')
+    }
+  }, [])
+
+  useEffect(() => {
+    if(movieVideos?.length) {
```

Comment:
what is the purpose of the ? on this line?

Reply:
I had originally put it there for conditional chainring of the .length in case it was undefined, to avoid a console error, but thinking about it further I don't think movieVideos would ever be undefined, maybe just an empty array - I can change this as well as remove the unnecessary videos in the fixture